Gio-2.0.gir

```diff
@@ -74200,6 +74213,22 @@ Note that the private key is not accessed until usage and may fail or require a
           </parameter>
         </parameters>
       </constructor>
+      <constructor name="new_from_pkcs12" c:identifier="g_tls_certificate_new_from_pkcs12" throws="1">
+        <return-value transfer-ownership="full">
+          <type name="TlsCertificate" c:type="GTlsCertificate*"/>
+        </return-value>
+        <parameters>
+          <parameter name="data" transfer-ownership="none">
```

Comment:
Same problems here. No version, and the `length` annotation for the array was ignored but is in the code